index.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express')
+var bodyParser = require('body-parser')
+
+var app = express()
+
+app.use(bodyParser.json())
+app.use(express.static('public'))
+
+app.get('/', function(req, res){
+    res.sendFile('index.html')
+})
+
+
+app.use("/api/student", require('./app/index'))
+
+app.listen(3000, console.log('Server started'))
```

Comment:
This is missing to inform the address and port in console message and make it easy for whom is starting the application!